packages/admin/cms-admin/src/pages/pagesPage/PagesPage.tsx

```diff
@@ -179,18 +179,27 @@ export function PagesPage({
                                 />
                             </ActionToolbarBox>
                             <FullHeightPaper variant="outlined">
-                                {loading ? (
+                                {loading && isInitialLoad.current ? (
                                     <Loading behavior="fillParent" />
                                 ) : (
-                                    <PageTree
-                                        ref={refPageTree}
-                                        pages={pagesToRenderWithMatches}
-                                        editDialogApi={editDialogApi}
-                                        toggleExpand={toggleExpand}
-                                        onSelectChanged={onSelectChanged}
-                                        category={category}
-                                        siteUrl={siteConfig.url}
-                                    />
+                                    <>
+                                        <Divider />
+                                        {loading && !isInitialLoad.current ? (
+                                            <LinearProgress sx={{ height: 2 }} />
+                                        ) : (
+                                            /* Placeholder to avoid content jumping when the loading bar appears */
+                                            <Box sx={{ backgroundColor: "white", width: "100%", height: 2 }} />
```

Comment:
What do you mean? I moved the divider to have the border above the LinearProgress/ Placeholder. Didn't want to pass two props just for this purpose.

Reply:
The blue I used is just the defined primary color 🤔

Reply:
> What do you mean?

Couldn't it be `loading && !isInitialLoad.current ? <LinearProgress /> : <Divider />` instead of always rendering the divider and a blank placeholder? 🤔

Reply:
I just tested it locally: The divider's height is 1px, but the linear progress is 2x. So we would have to add a blank placeholder anyway.